doc/metadata.md

```diff
@@ -46,7 +44,7 @@
 
     struct dirent {
         u64 ino;
-        string name;
+        char name[PATH_MAX];
```

Comment:
is binary search used also on dirents or could we use a variable length string?

Reply:
No, this could definitely be variable length. Let me define a string type and reference that here (and above).